### DISCUSSION TITLE
API for getting all active sessions?

### DISCUSSION THREAD

**Discussion by tamx** [bluesky-social/atproto#903] [Clarifications and "Why?"]
I don't see a way to get a list of active sessions. Is there a reason for this? For security reasons, I believe you should revoke sessions you don't know.

**Comment by ianklatzco** [comment]
There are two tokens: `access` and `refresh`. 

Access tokens last for 120 minutes.

Refresh tokens last for 90 days.

https://github.com/bluesky-social/atproto/blob/a832c54de7d94e10b0fc8ac2c17746bd58190551/packages/pds/src/auth.ts#L44-L68

It looks like there might not even be a way to change your password exposed in the UI currently!

**Reply by ianklatzco** [reply]
There is a `getSession`:
https://github.com/bluesky-social/atproto/blob/ece4b61d50543dc145c52265a77ad47ae36cb09d/lexicons/com/atproto/server/getSession.json

**Reply by tamx** [reply]
Maybe it's the current session and I think it's different from the intention of this Q&A.
